src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -589,6 +597,11 @@ private IEnumerable<string> LibrarySearchPaths(string library)
             yield return Path.GetFullPath(library);
         }
 
+        internal static string GetPythonTemplatePath()
```

Comment:
@radumg thanks for looking into how to avoid the creation of a new `pathManage`r, but this smells funny to me, and it is a bit inconsistent with other preferences which are similar, like the customPackageFolder paths.

I'm thinking we could follow a similar pattern like here:
https://github.com/DynamoDS/Dynamo/blob/7304ba11089f68d50b8e8ca120d23eaa1b988609/src/DynamoCore/Models/DynamoModel.cs#L583

so we would assign the default path to `""` in the preferences constructor and instead set it to path specified in the `pathManager` during DynamoModel creation time.

Let me know what you think of this - both seem okay on their own, but this seems a more consistent solution and we can avoid the static property on `pathManager`

Reply:
Thanks @mjkkirschner - think your suggestion is good, it's indeed inconsistent with rest of props. I'll go ahead and implement the above, was just trying to avoid modifying the `DynamoModel` class/ctor as per our previous conversations.

Reply:
Ok, I've implemented this in c4bff1548ee25b487e255e5b915bbcecf9abfe51